src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts

```diff
@@ -12,6 +12,18 @@ import { InstructorHelpSectionComponent } from '../instructor-help-section.compo
 })
 export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSectionComponent implements OnInit {
 
+  isEssayQsCollapsed: boolean = false;
+  isMCQSingleCollapsed: boolean = false;
+  isMCQMultCollapsed: boolean = false;
+  isNumScaleCollapsed: boolean = false;
+  isPtsOptionsCollapsed: boolean = false;
+  isPtsRecipientsCollapsed: boolean = false;
+  isContribQsCollapsed: boolean = false;
+  isRubricQsCollapsed: boolean = false;
+  isRankOptsCollapsed: boolean = false;
+  isRankRcptsCollapsed: boolean = false;
```

Comment:
```suggestion
  isRankOptionsCollapsed: boolean = false;
  isRankRecipientsCollapsed: boolean = false;
```

Might be worthwhile to make these more verbose, could  easily be misread considering that you utilised "Pts" to mean "Points" earlier.

Reply:
Done.